contracts/ERC721A.sol

```diff
@@ -405,6 +405,38 @@ contract ERC721A is Context, ERC165, IERC721, IERC721Metadata {
         _afterTokenTransfers(address(0), to, startTokenId, quantity);
     }
 
+    /**
+     * @dev Equivalent to _mint(to, quantity, '', false).
+     */
+    function _mint(address to, uint256 quantity) internal {
```

Comment:
Should we just rename the existing mint function to _safeMint and remove the `safe` parameter? This will be backwards incompatible, but will make the code cleaner since I don't see ppl calling `_mint(to, qty,'', false)` going forward

Reply:
Agree to remove the `safe` parameter and rename `_safeMint`. 
For backward compatibility, we could add back `_mint(to, qty, data, safe)` and then call `_mint` and `_safeMint` based on the `safe` parameter, but it cost maintainability.

Reply:
I’m just worried about the customers calling the ERC721A helpdesk if we break backwards compatibility. Lemme think for a while.

Reply:
Well we've broken it a couple times already. I'm ok cutting a v4 of ERC721A for a change like that

Reply:
Outside rn, brb in a few hours probably

Reply:
Code is neater, about 50 gas saved. Let’s go.